Week-4/Assignment_4.2/exercise_4.2.js

```diff
@@ -0,0 +1,10 @@
+class Person {
+  constructor(name) {
+    this.name = name;
+  }
+}
+const teacher = new Person("Nachiket");
```

Comment:
Teacher has to derived from Person and teach function should be a prototype of teacher